package.json

```diff
@@ -4,14 +4,10 @@
   "repository": "https://github.com/barnardos/design-system.git",
   "private": true,
   "scripts": {
-    "check-format":
-      "prettier '**/*.{css,json,js,md}' '!package.json' '!.github/*.md' --list-different --ignore-path .gitignore",
-    "format":
-      "prettier '**/*.{css,json,js,md}' '!package.json' '!.github/*.md' --write --ignore-path .gitignore",
-    "lint:css":
-      "stylelint '**/*.css' --max-warnings=0 --ignore-path .gitignore",
-    "lint:md":
-      "remark . --quiet --frail --rc-path .remarklintrc.json --ignore-path .gitignore",
+    "check-format": "prettier '**/*.{css,json,js,md}' '!package.json' '!.github/*.md' --list-different --ignore-path .gitignore",
```

Comment:
Every time I edit `package.json`, prettier reformats it to drop all the values to the next line. How can we get around these changes if they are to be inline?

Reply:
This reverts it to the correct format (i.e. the one autoformatted by yarn). Going forward we need to find a way to configure the VSCode prettier extension to ignore this file.